src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -49,6 +50,22 @@ def _get_value_from_message_dict(
     return d
 
 
+class DisallowedFilePurpose(ValueError):
+    """Is raised when the specified file purpose is not in allowed values."""
```

Comment:
```suggestion
    """Provides the error raised when the specified file purpose is not in the allowed values."""
```
Descriptions for classes should begin with a verb ending in "s" or "es" for consistency.

Reply:
@seanpearsonuk @PipKat 

Yes, Thanks for the suggestion. 

I agree that descriptions for classes should begin with a verb ending in "s" or "es". We have followed this guideline in all custom exception classes. ("Provides the error...")

I think we are following this guideline in current case as well because "Is" also a supporting _verb_ ending with "s". 

We are planning to change all docstrings from "Provides the error when the..." to "Is raised when the..." because we think that "Is raised..." is more appropriate.

@PipKat  Could you please confirm that whether "Is raised..." follows documentation guideline or not, because we want to update all docstrings in new format ("Is raised...") and we also want to follow documentation guidelines.

Thank you.

Reply:
The description begins _Is_ which is a verb ending with s, so there is no issue.

Reply:
@PipKat We will avoid gratuitous use of _is_, but it works very well in this narrow case. The doc-string reflects exactly how you would explain the class to someone conversationally whereas _provides the error_ does not.